lasagne/tests/layers/test_normalization.py

```diff
@@ -101,8 +101,8 @@ def input_layer(self, input_data):
     @pytest.fixture
     def layer(self, input_layer):
 
-        from lasagne.layers.normalization import\
-                LocalResponseNormalization2DLayer
+        from lasagne.layers.normalization import (
+            LocalResponseNormalization2DLayer,)
```

Comment:
Same here. (Although the new version may be better, it shouldn't be part of this PR.)